app/presenters/register_api/single_application_presenter.rb

```diff
@@ -0,0 +1,245 @@
+module RegisterAPI
+  class SingleApplicationPresenter
+    UCAS_FEE_PAYER_CODES = {
+      'SLC,SAAS,NIBd,EU,Chl,IoM' => '02',
+      'Not Known' => '99',
+    }.freeze
+
+    def initialize(application_choice)
+      @application_choice = ApplicationChoiceExportDecorator.new(application_choice)
+      @application_form = application_choice.application_form
+    end
+
+    def as_json
+      {
+        id: application_choice.id.to_s,
+        type: 'application',
+        attributes: {
+          support_reference: application_form.support_reference,
+          status: status, # keep to show offer withdrawn and deferred
+          updated_at: application_choice.updated_at.iso8601,
+          submitted_at: application_form.submitted_at.iso8601,
+          recruited_at: application_choice.recruited_at.iso8601,
```

Comment:
good catch adding it here! I reckon we have to fix this in the vendor API as well